lib/hero-image-header/index.js

```diff
@@ -1,6 +1,8 @@
 import React, { Component, PropTypes } from 'react';
 require('./style.css');
 const signpost = require('../../src/assets/goingplaces.svg');
+const bouncingArrow = require('../../src/assets/down-arrow.svg');
```

Comment:
didn't realise you *could* `require` an `.svg` file... presumably webpack enables this *magic*? 🎩